app/components/events-section.tsx

```diff
@@ -0,0 +1,70 @@
+export default function EventsSection() {
+  return (
+    <section className="max-w-screen-xl bg-[#E4EFF0]">
+      <div className="sm:ml-24 pt-10 pb-20 space-y-3 ">
+        <h2 className="text-center sm:text-left sm:text-5xl text-4xl font-bold italic text-[#1F2937]">
```

Comment:
Feel free to add `#1F2937` to our Tailwind config so that we can reference it by a constant name!